src/target/riscv/riscv-013.c

```diff
@@ -1874,7 +1882,16 @@ void riscv013_reset_current_hart(struct target *target)
 	control = set_field(control, DMI_DMCONTROL_NDMRESET, 0);
 	dmi_write(target, DMI_DMCONTROL, control);
 
```

Comment:
This may take several seconds to come out of reset. I think there are  parameters you can specify in your OpenOCD CFG file that you should use here. Otherwise you may unnecessarily start bumping up the busy delay, which only ever goes up.